src/qibolab/runcards/iqm5q.yml

```diff
@@ -4,9 +4,9 @@ description: IQM 5-qubit device, controlled with Zurich Instruments.
 settings:
     nshots: 4096
     sampling_rate: 2.e+9
-    relaxation_time: 300.e-6
-    time_of_flight: 280.e-9
-    smearing: 0.e-9
+    relaxation_time: 300_000
```

Comment:
AT LEAST PUT IT IN SCIENTIFIC NOTATION !

Reply:
Meditation